db.go

```diff
@@ -464,11 +464,7 @@ func (db *DB) getMemTables() ([]*skl.Skiplist, func()) {
 		immt.IncrRef()
 		tables = append(tables, immt)
 	}
-	return tables, func() {
-		for _, tbl := range tables {
-			tbl.DecrRef()
-		}
-	}
+	return tables
```

Comment:
This change seems don't reduce lock scope.

Reply:
It return tables explicitly, let the caller to do clean up staff.